smsender/src/main/java/fr/iolabs/leaf/smsender/services/SmsServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package fr.iolabs.leaf.smsender.services;
+
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SmsServiceImpl implements SmsService{
```

Comment:
We do not split interface from Impl in leaf for now.
No need to split in two classes.